main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+
+	. "github.com/jacobsa/ogletest"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+
+func TestMains(t *testing.T) { RunTests(t) }
```

Comment:
why is it plural? rename it to TestMain

Reply:
It is sawing error "Wrong test signature"